dataplane/saiserver/acl.go

```diff
@@ -232,6 +233,13 @@ func (a *acl) createAclEntryFields(req *saipb.CreateAclEntryRequest, id uint64,
 			Masks:   binary.BigEndian.AppendUint16(nil, uint16(req.GetFieldEtherType().GetMaskUint())),
 		})
 	}
+	if req.GetFieldIcmpType() != nil {
+		aReq.EntryDesc.GetFlow().Fields = append(aReq.EntryDesc.GetFlow().Fields, &fwdpb.PacketFieldMaskedBytes{
+			FieldId: &fwdpb.PacketFieldId{Field: &fwdpb.PacketField{FieldNum: fwdpb.PacketFieldNum_PACKET_FIELD_NUM_ICMP_TYPE}},
```

Comment:
The one at line 245 is also PacketFieldNum_PACKET_FIELD_NUM_ICMP_TYPE. Is it applied to both V4 and V6?

Reply:
yeah